src/utils/download-file.js

```diff
@@ -9,25 +14,5 @@ export default async function downloadFile( response ) {
 	const filename = response.headers
 		.get( 'Content-Disposition' )
 		.split( 'filename=' )[ 1 ];
-
-	// Check if the browser supports navigator.msSaveBlob or navigator.saveBlob
-	if ( window.navigator.msSaveBlob || window.navigator.saveBlob ) {
-		const saveBlob =
-			window.navigator.msSaveBlob || window.navigator.saveBlob;
-		saveBlob.call( window.navigator, blob, filename );
-	} else {
-		// Fall back to creating an object URL and triggering a download using an anchor element
-		const url = URL.createObjectURL( blob );
-
-		const a = document.createElement( 'a' );
-		a.href = url;
-		a.download = filename;
-		document.body.appendChild( a );
-		a.click();
-		document.body.removeChild( a );
-
-		setTimeout( () => {
-			URL.revokeObjectURL( url );
-		}, 100 );
-	}
+	downloadBlob( filename, blob, 'application/zip' );
```

Comment:
This is a minor refactoring. With this change, we no longer check if the browser is IE, but this should not be a problem since WordPress itself does not support IE.

Reply:
I was actually unfamiliar with `msSaveBlob` until looking into this change.  Likewise WordPress' implementation of the anchor downloading technique so I'm happy to replace that.  It works as expected.

Thank's for cleaning that up.